cirq-core/cirq/ops/tags.py

```diff
@@ -43,3 +43,26 @@ def _json_dict_(self) -> Dict[str, str]:
 
     def __hash__(self):
         return hash(VirtualTag)
+
+
+class RoutingSwapTag:
+    """A 'cirq.TaggedOperation' tag indicated that the operation is an inserted SWAP.
+    An InsertedSwapTag is meant to be used internally to distinguish SWAP operations that are
```

Comment:
nit: Insert newline between docstring summary and detailed body. Also, update docstring (it's not InsertedSwapTag anymore and not an internal class anymore)

Reply:
done